packages/statemanager/src/stateManager.ts

```diff
@@ -329,6 +465,9 @@ export class DefaultStateManager extends BaseStateManager implements StateManage
    */
   async getProof(address: Address, storageSlots: Buffer[] = []): Promise<Proof> {
     const account = await this.getAccount(address)
+    if (!account) {
```

Comment:
Here again, very surprised the linter isn't complaining.  I thought we had a rule for explicit boolean checks where a variable contains either a value or `undefined`?
```suggestion
    if (account === undefined) {
```